hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/AutoCreatedQueueTemplate.java

```diff
@@ -170,8 +170,7 @@ private void setTemplateConfigEntries(CapacitySchedulerConfiguration configurati
     int wildcardLevel = 0;
     // root can not be wildcarded
     // MAX_WILDCARD_LEVEL will be configurable in the future
-    int supportedWildcardLevel = Math.min(queuePathMaxIndex - 1,
-        MAX_WILDCARD_LEVEL);
+    int supportedWildcardLevel = Math.min(queuePathMaxIndex, MAX_WILDCARD_LEVEL);
```

Comment:
Can you fix this issue in a separate jira? We might need to backport this change separately.

Reply:
https://issues.apache.org/jira/browse/YARN-11071

Done, I'll create a PR for that and rebase this branch on it.